src/js/main.js

```diff
@@ -158,10 +156,29 @@ $(document).ready(function() {
 
   //Display the active filter
   $('#filter ul button').on('click', function(event) {
+    $element = $(this);
+    if ($element.hasClass('selected')) {
+      console.log('Jan surprise!');
```

Comment:
Hmm let's not leave `console.log()`s in public space. These are only for debugging, not for production. Please remove.